homeassistant/components/mjpeg/config_flow.py

```diff
@@ -54,7 +54,7 @@ def async_get_schema(
 
     if show_name:
         schema = {
-            vol.Optional(CONF_NAME, default=defaults.get(CONF_NAME)): str,
+            vol.Required(CONF_NAME, default=defaults.get(CONF_NAME)): str,
```

Comment:
Shouldn't we fix the default instead so that it's never `None`?

Also, we should not use `dict.get` on line 157 since we know that the key will exist in the dict.

Reply:
Agree. To be honest, I don’t understand why the MJPEG variant of Generic Camera exists at all.